### ISSUE TITLE
SHA256 of latest release has changed - plantuml-1.2023.8.jar

### ISSUE THREAD

**Issue by gibfahn** [plantuml/plantuml#1447]
**Describe the bug**

The sha256sum 2 weeks ago for https://github.com/plantuml/plantuml/releases/download/v1.2023.8/plantuml-1.2023.8.jar was `4d4084ce85dbb1072fb2cfeae5100ef3a7b712e17b54b87eb612593e74d62f1f` , now it is `0a200db6e485c18206fc98e3ba1b44876c31bcec167ec0d5f152eb67c87aad46`.

You can see this from the runs at https://github.com/Homebrew/homebrew-core/pull/131526 and https://github.com/Homebrew/homebrew-core/pull/133037.

Systems like brew assume that released artifacts are immutable (for security purposes), could you confirm that this change was intentional, and that the new artifact is the one to use?

**To Reproduce**

```console
❯ brew fetch -s --formula Formula/plantuml.rb
==> Downloading https://github.com/plantuml/plantuml/releases/download/v1.2023.8/plantuml-1.2023.8.jar
Warning: Formula reports different sha256: 4d4084ce85dbb1072fb2cfeae5100ef3a7b712e17b54b87eb612593e74d62f1f
Already downloaded: /Users/gib/Library/Caches/Homebrew/downloads/fd6842251850d5bfc5138b2106b024c8f9110487b2c73438c664317d917090eb--plantuml-1.2023.8.jar
SHA256: 0a200db6e485c18206fc98e3ba1b44876c31bcec167ec0d5f152eb67c87aad46
```

**Expected behavior**

Published releases are immutable, and the sha256sums don't change.

**Comment by arnaudroques** [comment]
> Published releases are immutable, and the sha256sums don't change.

Yes, sorry about that.
We have some difficulties on publishing multi-license versions (MIT, BSD, GPL...) so we have to publish several times 1.2023.8
It's really the same code, I assume that SHA256 changes because some timestamps may be different.

> could you confirm that this change was intentional, and that the new artifact is the one to use?

Yes, I confirm.

**Comment by chenrui333** [comment]
We are addressing this checksum mismatch in the formula version bump, https://github.com/Homebrew/homebrew-core/pull/133691 

This issue can be closed now. Thanks for confirming the git re-tagging is intended.

**Comment by arnaudroques** [comment]
We have just published V1.2023.9
Please reopen this issue (or a new one) if ever it happens again: it should not now.
Thanks!

**Comment by gibfahn** [comment]
Thanks for the clarification!
